wgengine/magicsock/magicsock.go

```diff
@@ -1622,7 +1625,12 @@ func (c *Conn) ReceiveIPv4(b []byte) (n int, ep conn.Endpoint, err error) {
 			}
 			return 0, nil, err
 		}
-		if ep, ok := c.receiveIP(b[:n], pAddr.(*net.UDPAddr), &c.ippEndpoint4); ok {
+		pAddr, _ = addr.(*net.UDPAddr)
```

Comment:
why comma ok? I'd rather have an explicit crash on this line than a nil ptr deref on the next?

Reply:
Ack. Deleted in the subsequent commit, so going to leave here.